scripts/report_scripts/taxonomic_classification.Rmd

```diff
@@ -27,58 +28,88 @@ suppressMessages(suppressWarnings(library(DT)))
 library(knitr)
 ```
 
-This metagenomic analysis report provides an overview of the taxa that were classified from the set of unaligned reads 
-of the given sample. The analysis of unaligned reads can provide insight about the alignment quality, cleanliness 
-of the sample, possible species with high sequence similarity which could introduce alignment bias and more.  
-<font size="3"> *Note: Because of algorithmic differences between the classification and the sequence alignment, a high 
-number of fragments assigned to the species of interest (in this case SARS-CoV-2) could be a result of differences in 
-thresholds for sequence similarity. In this case, further adjustment of the alignment tool and/or the classifier are
+This metagenomic analysis report provides an overview of the taxa that were 
```

Comment:
Why does this paragraph look so jagged?

Reply:
Here I was just lazy with enforcing the line width limit, inserting just spaces and not adjusting the rest of the line. I will fix this in the coming PR for general fixes.

Reply:
If you're using Emacs you can reflow the paragraph with `M-q`.